packages/plugins/package.json

```diff
@@ -65,7 +64,7 @@
     "typescript": "^5.1.3",
     "yargs": "^17.7.2",
     "yesno": "^0.4.0",
-    "hardhat": "2.20.1"
+    "hardhat": "2.22.3"
```

Comment:
This fixes [CHU-1067](https://linear.app/chugsplash/issue/CHU-1067/deployments-on-anvil-fail-due-to-blob-gas-issue-in-simulation).